From 115c316313044595af1ba5b1d60a435d2f248dac Mon Sep 17 00:00:00 2001
Message-ID: <115c316313044595af1ba5b1d60a435d2f248dac.1726652909.git.mjg@fedoraproject.org>
From: Michael J Gruber <mjg@fedoraproject.org>
Date: Wed, 18 Sep 2024 11:47:03 +0200
Subject: [PATCH] tests/: test_2608(): (red)add expected result

9ccf1c38 ("tests/: test_2608(): expect different output with mupdf master.", 2024-09-17)
changed the result for master but meant to leave in the 1.24.x result.
Re-add the file from that commit at the newly expected location.
---
 tests/resources/test_2608_expected-1.24.x | 10 ++++++++++
 1 file changed, 10 insertions(+)
 create mode 100644 tests/resources/test_2608_expected-1.24.x

diff --git a/tests/resources/test_2608_expected-1.24.x b/tests/resources/test_2608_expected-1.24.x
new file mode 100644
index 00000000..d550f123
--- /dev/null
+++ b/tests/resources/test_2608_expected-1.24.x
@@ -0,0 +1,10 @@
+No significant gamma-ray excess above the expected background
+is detected from the direction of FRB 20171019A, with 52 gamma
+candidate events from the source region and 524 background event.
+A second analysis using an independent event calibration and reconstruction (Parsons & Hinton 2014) confirms this result. A search for
+variable emission on timescales ranging from milliseconds to several minutes with tools provided in (Brun et al. 2020) does not reveal
+any variability above 2.2 ùúé. For the total data set of 1.8 h, 95% confidence level (C. L.) upper limits on the photon flux are derived using
+the method described by Rolke et al. (2005). The energy threshold
+of the data is highly dependent on the zenith angle of the observations. For these observations, the zenith angles range from 15 to 25
+deg, which leads to an energy threshold for the stacked data set of
+ùê∏th = 120 GeV. The upper limit on the Very High Energy (VHE)
-- 
2.46.1.784.gb456aa5b01

